lms/templates/learner_dashboard/certificate.underscore

```diff
@@ -1,7 +1,7 @@
 <div class="certificate-container">
-    <p class="title"><%- gettext('XSeries Program Certificates') %>:</p>
+    <h2 class="hd-6"><%- gettext('XSeries Program Certificates') %>:</h2>
```

Comment:
I think this is one of the benefits of a Pattern Library though, because it means we can consistently style elements using these already defined classes. Meaning less time has to be spent writing Sass (which would just be an alias for the same class anyway).